src/merge_tools.rs

```diff
@@ -451,7 +453,15 @@ fn get_diff_editor_from_settings(
         CommandNameAndArgs::String(name) => get_tool_config(settings, name)?,
         CommandNameAndArgs::Vec(_) => None,
     };
-    Ok(maybe_editor.unwrap_or_else(|| MergeTool::with_edit_args(&args)))
+    let mut editor = maybe_editor.unwrap_or_else(|| MergeTool::with_edit_args(&args));
+    // If no interpolation parameters found, fall back to "$left $right"
+    // (TODO: or error out like merge-args?)
+    if !editor.edit_args.iter().any(|arg| arg.contains('$')) {
```

Comment:
Maybe check for $left and $right specifically, or at least for `\$\w+` regex?

Reply:
Explicit check would introduce another problem like "what if only `$left` is specified", which I really want to avoid.

Perhaps, we can instead make `merge-tools.<tool>.edit-args` default to `$left $right`.
It's a breaking change, but only affects to those who customize `merge-tools`. For any `ui.diff-editor = <name>` not defined in the `merge-tools` table, the default `$left $right` will be used. No ambiguity.

And that's the reason why I updated stock `merge-tools` configs.

Reply:
> Perhaps, we can instead make merge-tools.<tool>.edit-args default to $left $right

This sounds good to me, it does seem to make things more consistent.

Reply:
Updated to fill edit-args by default, instead of appending.